src/modules/vendor/components/courses/ProfileFollowUp.vue

```diff
@@ -181,6 +180,18 @@ export default {
       const type = this.messageTypeOptions.find(type => type.value === value);
       return type ? type.label : '';
     },
+    courseNotStartedYet () {
+      const slots = this.course.slots.filter(slot => this.$moment().isAfter(slot.endDate))
+      return !slots.length;
+    },
+    loadMessageTypeOptions () {
+      this.messageTypeOptions.length = 0;
```

Comment:
Bien vu ! 
Parcontre on utilise plutot la syntaxe: `this.messageTypeOptions = [];`